src/main/java/aspects/LockOnUsername.java

```diff
@@ -0,0 +1,27 @@
+package aspects;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.codehaus.jackson.map.RuntimeJsonMappingException;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by benrudolph on 10/19/16.
+ */
+@Component
+@Aspect
+public class LockOnUsername {
+
+    @Around("@annotation(annotations.LockAndBlock)")
```

Comment:
this should run around any method that has the `@LockAndBlock` annotation, but it's not currently working